### ISSUE TITLE
drudgereport.com: breakage

### ISSUE THREAD

**Issue by battaglia01** [uBlockOrigin/uAssets#18519]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/124740436/235392297-c4fc6290-73a6-4a70-96a9-3f6a7683f21d.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://drudgereport.com/`

### Category

breakage

### Description

Breaks layout of Drudge Report on uBlock Origin on Firefox Mobile Android; makes it render similarly to the desktop mode. This is a brand new install so there are no custom filters or stale cached filter lists. No other add ons were enabled. Pictures below.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

With uBlock on
![Screenshot_20230615-121354_Firefox](https://github.com/uBlockOrigin/uAssets/assets/1020594/eac008c3-3fbe-4a94-bde8-2309c2e1c773)

With uBlock off
![Screenshot_20230615-121420_Firefox](https://github.com/uBlockOrigin/uAssets/assets/1020594/47f32f66-9803-40d4-93d2-98f2bcfb9c46)

</details>

### Configuration

<details>

```yaml
uBlock Origin 1.50.0:
 since: 6m
Firefox Mobile: 114
filterset (summary):
 network: 95514
 cosmetic: 29245
 scriptlet: 17729
 html: 942
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 34768-410, 5m
  ublock-badware: 7351-0, 5m
  ublock-privacy: 439-1, 5m
  ublock-unbreak: 2067-1, 5m
  ublock-quick-fixes: 275-1, 5m
  adguard-mobile: 8282-122, 5m
  easylist: 66540-17254, 5m
  easyprivacy: 30867-666, 5m
  urlhaus-1: 7714-0, 5m
  plowe-0: 3738-2, 5m
filterset (user): [empty]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 2519 ms
 maxAssetCacheWait: 16 ms
popupPanel:
 blocked: 16
 network:
  drudgereport.com: 8
  cloudflare.com: 2
  d185y8fkx3tp2z.cloudfront.net: 2
  googletagservices.com: 2
  rubiconproject.com: 2
```
</details>

**Comment by mapx-** [comment]
add

```
||drudgereport.com/js/responsive-$badfilter
@@||ads.rubiconproject.com/prebid/$script,domain=drudgereport.com
```

@ryanbr EL + EP issues

**Comment by battaglia01** [comment]
Thanks @mapx and @ryanbr. I have updated lists and responsiveness seems to work correctly on my computer, in Chrome, in the inspector running in device mode. However, I can't seem to get it to work on my actual Android device. I've clicked "purge all caches" and "Update now," cleared my device history, cookies, etc, and even uninstalled uBlock Origin and reinstalled it again and it still isn't working on the Android device. (I am not sure how to hold the shift key when clicking "purge all caches" on mobile, but surely uninstalling uBlock Origin and reinstalling should clear any kind of cache.)
